compute_endpoint/tests/unit/test_endpoint_config.py

```diff
@@ -1,7 +1,15 @@
+import random
 import typing as t
 
 import pytest
 from globus_compute_endpoint.endpoint.config.model import ConfigModel
+from globus_compute_sdk.sdk.utils import AMQP_PORT_TYPE
+
+try:
+    from typing import get_args
+except ImportError:
+    # for 3.7
+    from typing_extensions import get_args
```

Comment:
I recommend the following syntax, which `pyupgrade` is able to recognize and auto-rewrite throughout the project when Python 3.7 and lower are no longer supported:

```suggestion
if sys.version_info >= (3, 8):
    from typing import get_args
else:
    # for 3.7
    from typing_extensions import get_args
```